contrib/AdaEurope2018/derivation_count.adb

```diff
@@ -5,17 +5,19 @@ with Ada.Containers.Hashed_Maps;
 with Ada.Text_IO;
 
 with Libadalang.Analysis;
+with Libadalang.Common;
 
 with Helpers;
 
 procedure Derivation_Count is
 
    package TIO renames Ada.Text_IO;
    package LAL renames Libadalang.Analysis;
-   use type LAL.Ada_Node_Kind_Type;
+   package LAC renames Libadalang.Common;
```

Comment:
Could you rename this to `LALCO`, for consistency with the rest of the codebase?

Reply:
Done